sentry/src/main/java/io/sentry/buffer/DiskBuffer.java

```diff
@@ -173,7 +174,11 @@ private Event getNextEvent(Iterator<File> files) {
      */
     @Override
     public Iterator<Event> getEvents() {
-        final Iterator<File> files = Arrays.asList(bufferDir.listFiles()).iterator();
+        final File[] fileArray = bufferDir.listFiles();
+        if (fileArray == null) {
+            return new ArrayList<Event>(0).iterator();
```

Comment:
have you tried Collections.emptyIterator() ?

Reply:
It's what I had when I called you. IDE broken and didn't figure out where to import it from

Reply:
😢